docs/docs/integrations/tools/goat.ipynb

```diff
@@ -0,0 +1,250 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "id": "afaf8039",
+   "metadata": {
+    "vscode": {
+     "languageId": "raw"
+    }
+   },
+   "source": [
+    "---\n",
+    "sidebar_label: GOAT\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e49f1e0d",
+   "metadata": {},
+   "source": [
+    "# GOAT\n",
+    "\n",
+    "## Overview\n",
+    "[GOAT](https://github.com/goat-sdk/goat) is the largest agentic finance toolkit for AI agents.\n",
```

Comment:
Could you remove marketing adjectives / sales language from doc-strings?

```suggestion
    "[GOAT](https://github.com/goat-sdk/goat) is the finance toolkit for AI agents.\n",
```

Reply:
Done!